src/encoded/tests/test_schema_genetic_modification.py

```diff
@@ -328,3 +348,24 @@ def test_tale_replacement_properties(testapp, tale_replacement, source):
     tale_replacement.update({'introduced_sequence': 'TTATCGATCGATTTGAGCATAGAAATGGCCGATTTATATGCCCGA'})
     res = testapp.post_json('/genetic_modification', tale_replacement, expect_errors=True)
     assert res.status_code == 201
+
+
+def test_mpra_properties(testapp, mpra):
```

Comment:
Not a big deal

- I normally see asserts at the end rather at the middle.
- If you need multiple asserts, that may be a sign you need multiple unit test. A unit test supposed to test a single functionality (unit) and not multiple actions.

Reply:
This branch branches off ENCD-4464-add-GM-category-enums, so that’s not my code. I only modified genetic_modification.js